superset/assets/javascripts/dashboard/v2/util/propShapes.jsx

```diff
@@ -19,6 +19,9 @@ export const componentShape = PropTypes.shape({ // eslint-disable-line
     text: PropTypes.string,
     headerSize: PropTypes.oneOf(headerStyleOptions.map(opt => opt.value)),
 
+    // slice id for chart holder
+    chartKey: PropTypes.string,
```

Comment:
please just use the slice id and call it `id`. rule of thumb: most of the time you don't need comments if your variables have descriptive names!